src/status_im/contexts/chat/messenger/messages/content/view.cljs

```diff
@@ -126,14 +126,62 @@
       :photo-path          photo-path
       :incoming?           (not= public-key from)}]))
 
+(defn group-member
+  [literal]
+  (let [[primary-name _] (rf/sub [:contacts/contact-two-names-by-identity literal])
+        photo-path       (rf/sub [:chats/photo-path literal])]
+    [rn/view {:style style/group-member-mention}
+     [quo/user-avatar
+      {:full-name         primary-name
+       :profile-picture   photo-path
+       :status-indicator? false
+       :size              :xxxs
+       :ring?             false}]
+     [quo/text
+      {:weight :semi-bold
+       :style  {:margin-left 4}}
+      primary-name]]))
+
+
+(defn resolve-group-system-message
+  [{:keys [^js children]} acc]
+  (conj acc
```

Comment:
The code could be a bit clearer if we avoided shadowing the outer `acc`. You could do that by not passing the `acc` to `resolve-group-system-message`.

It would look like this:

```clojure
(defn resolve-group-system-message
  [{:keys [children]}]
  (reduce (fn [acc {:keys [type literal]}]
            (if (= type "mention")
              (conj acc [group-member literal])
              (conj acc [rn/view [quo/text literal]])))
          [:<>]
          children))

(defn render-parsed-text
  [content]
  (reduce (fn [acc e]
            (conj acc (resolve-group-system-message e)))
          [quo/text]
          (:parsed-text content)))
```